playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -792,6 +792,8 @@ edxapp_generic_auth_config:  &edxapp_generic_auth
   ANALYTICS_API_KEY:  "{{ EDXAPP_ANALYTICS_API_KEY }}"
   EMAIL_HOST_USER: "{{ EDXAPP_EMAIL_HOST_USER }}"
   EMAIL_HOST_PASSWORD: "{{ EDXAPP_EMAIL_HOST_PASSWORD }}"
+  RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS: "{{ EDXAPP_RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS}}"
```

Comment:
i'm confused. does EDXAPP_RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS and RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS exist prior to your change?

same question for *RETRY_ACTIVATION_EMAIL_TIMEOUT

i'm happy to do some grep'ing myself. we may want to talk a little more about this in person too